WebService/appsettings.ini

```diff
@@ -31,6 +31,16 @@ webservice_url = "${PCS_STORAGEADAPTER_WEBSERVICE_URL}"
 # Max time allowed for requests to the storage adapter before reporting an error
 webservice_timeout = 10000
 
+[AzureManagementService]
+# URL where the azure management api is available.
+webservice_url = "https://management.azure.com"
+
+# Max time allowed for requests to the azure management service before reporting an error.
+# Value in milliseconds
+webservice_timeout = 10000
```

Comment:
this is already defined in line 32

Reply:
this value is specific to AzureManagementService calls